### PR TITLE
`azurerm_stream_analytics_output_servicebus_queue`: add support for `authentication_mode`

### PR DESCRIPTION
**Pull request by jiaweitao001**
* Test result:
```
--- PASS: TestAccStreamAnalyticsOutputServiceBusQueue_authenticationMode (407.17s)
PASS
```

### CODE REVIEW

File: internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go [hunk 1]
```diff
@@ -415,7 +431,30 @@ resource "azurerm_stream_analytics_output_servicebus_queue" "import" {
...
+  serialization {
+    type     = "Json"
+    encoding = "UTF8"
+    format   = "LineSeparated"
+  }
+}
+`, template, data.RandomInteger)
+}
+
+func (r StreamAnalyticsOutputServiceBusQueueResource) template(data acceptance.TestData, identity string) string {
```
Comment (katbyte) [inline internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go:457]: identity is unused in this function?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go (hunk 1 above)
Comment (jiaweitao001) [reply internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go:457]: it's used on L435.

Comment (katbyte) [reply internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go:457]: life 435 doesn't exist in this function? 
image

Comment (jiaweitao001) [reply internal/services/streamanalytics/stream_analytics_output_servicebus_queue_resource_test.go:457]: Sorry, misunderstood previously. Added `identity` usage in template now.

**Review by katbyte — APPROVED** [review]
LGTM 🍖

**Pull request merged**